shared/search/user-input/index.desktop.js

```diff
@@ -91,7 +91,7 @@ class UserInput extends Component<Props, State> {
       !trim(this.props.usernameText) &&
       this.props.onEnterEmptyText
     ) {
-      this.props.onEnterEmptyText()
+      this.props.selectedSearchId ? this.props.onAddSelectedUser() : this.props.onEnterEmptyText()
```

Comment:
Thanks for the review!

It's necessary to use `selectedSearchId` here rather than `showingSearchSuggestions` 'cause they mean different things.  Here's two different UI states:

A: You open the search box and press enter

B: You open the search box and press the down arrow and press enter

This check's distinguishing A and B.  In B, `selectedSearchId` will be the username of the first recommendation, so we'll select them, and in A it'll be null so we'll close the search dialog.  `showingSearchSuggestions` would be true in both cases.

Does that make sense?

Reply:
Ah yeah that makes sense. I was mistaking `selectedSearchId` as the id for the result list, rather than the individual result. sounds good then!